dts/bindings/retained_mem/zephyr,retained-reg.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2023 Nordic Semiconductor ASA
+# Copyright (c) 2023 Bjarki Arge Andreasen
+# SPDX-License-Identifier: Apache-2.0
+
+description: Retained register based retained memory area.
```

Comment:
too late to "suggest edits" but there is a repeated word:

description: Register-based retained memory area.